client/me/purchases/vat-info/index.tsx

```diff
@@ -406,6 +411,14 @@ function useRecordVatEvents( {
 	}, [ fetchError, updateError, isUpdateSuccessful, reduxDispatch ] );
 }
 
+function InputWrapper( { isError, children }: { isError: boolean; children: React.ReactNode } ) {
+	return (
+		<div className={ isError ? 'vat-form__field-wrapper .error' : 'vat-form__field-wrapper' }>
```

Comment:
I think the `error` class is not supposed to have a period.

But also we should probably make it more specific since `error` would be a pretty easy style to accidentally get targeted by other CSS. How about BED form?

```suggestion
		<div className={ isError ? 'vat-form__field-wrapper vat-form__field-wrapper--error' : 'vat-form__field-wrapper' }>
```

Reply:
Since this updateError value is not returning an error, we'll need to fix this in a future PR. Removed from the styling for now as the cart handles validation errors with notifications already, so error handling is at least handled more widely than the individual fields.